types/google.visualization/index.d.ts

```diff
@@ -1112,9 +1112,11 @@ declare namespace google {
             alternatingRowStyle?: boolean;
             cssClassNames?: CssClassNames;
             firstRowNumber?: number;
+            frozenColumns?: number;
             height?: string;
             page?: string;
             pageSize?: number;
+            pagingButtons?: number | 'both' | 'prev' | 'next';
```

Comment:
Please add `auto` as a possible value for `pagingButtons`.
Reference: [Docs](https://developers.google.com/chart/interactive/docs/gallery/table#configuration-options)